packages/extension-ui/src/Popup/ImportSeed/index.tsx

```diff
@@ -48,14 +48,14 @@ function ImportSeed(): React.ReactElement {
         setIsBusy(true);
 
         createAccountSuri(name, password, account.suri, type, account.genesis)
-          .then(() => onAction('/'))
+          .then(() => setStep((currentStep) => currentStep + 1))
```

Comment:
Since we're in an external popup now, we move over to the _completion_ screen that auto-closes, instead of moving back to the root path.